### PR TITLE
Group Details page - add/remove members feature

### PR DESCRIPTION
**Pull request by lgalis**
Group Details page - add/remove members feature

https://projects.engineering.redhat.com/browse/SSP-651

Screenshot from 2019-09-09 12-19-19
Screenshot from 2019-09-09 12-20-03
Screenshot from 2019-09-09 12-20-08

### CODE REVIEW

File: src/smart-components/group/principal/principals.js [hunk 1]
```diff
@@ -1,58 +1,102 @@
...
     if (uuid) {
       fetchGroup(uuid).then((data) => {
-        setRows(createRows(data.value.principals, filterValue));
+        setPrincipals(data.value.principals);
       });
     }
   };
 
+  const setCheckedPrincipals = (checkedPrincipals) => {
+    console.log('DEBUG - checked Principals: ', checkedPrincipals);
```
Comment (Hyperkid123) [inline src/smart-components/group/principal/principals.js]: console log

File: src/smart-components/group/principal/principals.js [hunk 2]
```diff
@@ -1,58 +1,102 @@
...
-          onClick: (_event, _rowId, principal) =>
-            history.push(`/principals/edit/${principal.uuid}`)
-        },
         {
           title: 'Delete',
           style: { color: 'var(--pf-global--danger-color--100)'	},
-          onClick: (_event, _rowId, principal) =>
-            history.push(`/principals/remove/${principal.uuid}`)
+          onClick: (_event, _rowId, principal) => {
+            removeMembers([ principal.username ]).then(() => fetchData());
```
Comment (Hyperkid123) [inline src/smart-components/group/principal/principals.js]: function `removeMembers` is already calling `fetchData`. Then branch is redundant here.

File: src/smart-components/group/principal/principals.js [hunk 3]
```diff
@@ -63,21 +107,25 @@ const mapStateToProps = ({ groupReducer: { selectedGroup, isLoading }}) => ({
...
+  location: PropTypes.shape({
+    pathname: PropTypes.string
+  }),
   principals: PropTypes.array,
-  platforms: PropTypes.array,
   isLoading: PropTypes.bool,
   fetchGroup: PropTypes.func.isRequired,
+  removeMembersFromGroup: PropTypes.func.isRequired,
   uuid: PropTypes.string,
+  match: PropTypes.object,
```
Comment (Hyperkid123) [inline src/smart-components/group/principal/principals.js]: There is a lot of props that are never used.

File: src/smart-components/group/principal/principals.js (hunk 2 above)
Comment (lgalis) [reply src/smart-components/group/principal/principals.js]: Thanks, you are right, I added that twice - fixing.

**Pull request merged**